grid-proxy/docs/database.md

```diff
@@ -0,0 +1,11 @@
+# Database
+
+The grid proxy has access to a postgres database containing information about the tfgrid.\
```

Comment:
why there's `\` in every line?

Reply:
specific information for the database or the current required storage?

Reply:
> why the `\` in every line?

just to separate lines when rendering the md file.